test/helper.rb

```diff
@@ -51,6 +51,10 @@ def strategy
       end
     end
   end
+
+  def http_site
+    strategy.client.site.sub(/^https?/, 'http')
+  end
```

Comment:
@printercu This change seems unrelated/unnecessary to this PR, can you remove these please? (unless I missing why they are needed.)